service/src/main/java/com/epam/rft/atsy/service/impl/CandidateServiceImpl.java

```diff
@@ -114,11 +117,23 @@ public CandidateDTO getCandidateByApplicationID(Long applicationID) {
               candidateFilterRequest.getPageSize());
     }
 
+    String
+        name =
+        MoreObjects.firstNonNull(candidateFilterRequest.getCandidateName(), StringUtils.EMPTY);
+    String
+        email =
+        MoreObjects.firstNonNull(candidateFilterRequest.getCandidateEmail(), StringUtils.EMPTY);
+    String
+        phone =
+        MoreObjects.firstNonNull(candidateFilterRequest.getCandidatePhone(), StringUtils.EMPTY);
+    String
+        positions =
+        MoreObjects.firstNonNull(candidateFilterRequest.getCandiadtePositions(), StringUtils.EMPTY);
```

Comment:
Minor improvement suggestion: local variables (name, email, phone, position) can be final.